### ISSUE TITLE
Flux: train only specific layers

### ISSUE THREAD

**Issue by StableLlama** [kohya-ss/sd-scripts#1537]
As written at https://www.reddit.com/r/StableDiffusion/comments/1f523bd/good_flux_loras_can_be_less_than_45mb_128_dim/ it can be sufficient just to train `single_transformer_blocks.7.proj_out` to get a very good and very small character LoRA.

It would be great when I could do such a training with this training script.

And it would be even greater when that would also lower the VRAM requirement significantly for that.

**Comment by FurkanGozukara** [comment]
i am looking forward to this as well

**Comment by singaporeandurian** [comment]
looking forward to this as well. if we can get any hints on the functions we can modify to get this working I'm happy to look into it Kohya!

**Comment by agxagi** [comment]
Yes same here! this is a game changer! it will save millions of Gigabyte  out there!
